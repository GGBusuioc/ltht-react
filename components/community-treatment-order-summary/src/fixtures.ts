import {
  LypftCommunityTreatmentOrder,
  CarePlanStatusCode,
  PartialDateTimeKindCode,
  CarePlanIntentCode,
  Metadata,
} from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const CommunityTreatmentOrderOne: LypftCommunityTreatmentOrder = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|communityTreatmentOrder',
  metadata: mockMetadata,
  consentToTreat: 'Consent Not Given',
  legalStatus: {
    coding: [{ display: 'Approved', code: '3309007' }],
    text: 'Approved',
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  restrictions: 'No restrictions',
}

const CommunityTreatmentOrderTwo: LypftCommunityTreatmentOrder = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|communityTreatmentOrder',
  metadata: mockMetadata,
  consentToTreat: 'Consent Given',
  legalStatus: {
    coding: [{ display: 'Pending', code: '779007' }],
    text: 'Pending',
  },
  period: {
    start: {
      value: '2020-01-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  restrictions: 'No Restrictions',
}

export { CommunityTreatmentOrderOne, CommunityTreatmentOrderTwo }
