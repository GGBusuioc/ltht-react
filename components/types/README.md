# `ltht-types`

> TODO: description

## Usage

```
const lthtTypes = require('ltht-types');

// TODO: DEMONSTRATE API
```
