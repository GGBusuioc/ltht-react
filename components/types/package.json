{
  "name": "@ltht-react/types",
  "version": "0.0.1-alpha.1",
  "description": "> TODO: description",
  "author": "Jonny Smith <jonny.smith@digitaljs.com>",
  "homepage": "https://github.com/ltht-epr/ltht-react/tree/master/components/ltht-types#readme",
  "license": "MIT",
  "main": "lib/index.js",
  "typings": "lib/index.d.ts",
  "directories": {
    "lib": "lib",
    "test": "__tests__"
  },
  "files": [
    "lib"
  ],
  "publishConfig": {
    "registry": "https://registry.yarnpkg.com"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/ltht-epr/ltht-react.git"
  },
  "scripts": {
    "build": "yarn build:lib && yarn build:es && yarn build:definition",
    "build:lib": "rimraf lib && babel src -d lib --extensions '.ts,.tsx' --source-maps --config-file ../../babel.config.js",
    "build:es": "rimraf es && cross-env BABEL_ENV=es babel src -d es --extensions '.ts,.tsx' --source-maps --config-file ../../babel.config.js",
    "build:definition": "tsc --noEmit false --emitDeclarationOnly",
    "build:watch": "tsc --noEmit false --emitDeclarationOnly --watch",
    "type:check": "tsc"
  },
  "bugs": {
    "url": "https://github.com/ltht-epr/ltht-react/issues"
  },
  "gitHead": "4b363f052431bd35738693f2a5986caa24c841cf"
}
