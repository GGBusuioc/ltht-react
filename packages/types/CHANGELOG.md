# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## [0.0.2](https://github.com/ltht-epr/ltht-react/compare/@ltht-react/types@0.0.1...@ltht-react/types@0.0.2) (2020-01-06)

**Note:** Version bump only for package @ltht-react/types





## [0.0.1](https://github.com/ltht-epr/ltht-react/compare/@ltht-react/types@0.0.1-alpha.3...@ltht-react/types@0.0.1) (2020-01-03)

**Note:** Version bump only for package @ltht-react/types
