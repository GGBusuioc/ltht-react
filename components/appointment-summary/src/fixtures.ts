import { Encounter, EncounterStatusCode, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const AppointmentOne: Encounter = {
  id: '346ca927-67e0-e931-a2c7-115087226fg2|appointment',
  metadata: mockMetadata,
  type: [
    {
      text: 'See GP regarding sore throat',
      coding: [{ display: 'GP', code: '64706' }],
    },
  ],
  period: {
    start: {
      value: '2020-04-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  serviceProvider: {
    display: 'Beeston Village Surgery',
    typeName: 'GP',
  },
  status: EncounterStatusCode.Planned,
}

const AppointmentTwo: Encounter = {
  id: '941ca927-34e0-e911-a2c7-005087226fe4|appointment',
  metadata: mockMetadata,
  type: [
    {
      text: 'Chest x-ray for recent car accident',
      coding: [{ display: 'Hospital', code: '26706' }],
    },
  ],
  period: {
    start: {
      value: '2019-12-12T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  serviceProvider: {
    display: 'Chapel Allerton Hospital',
    typeName: 'Hospital',
  },
  status: EncounterStatusCode.Finished,
}

export { AppointmentOne, AppointmentTwo }
