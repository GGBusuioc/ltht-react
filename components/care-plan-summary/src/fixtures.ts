import { CarePlan, CarePlanStatusCode, PartialDateTimeKindCode, CarePlanIntentCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const CarePlanOne: CarePlan = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|careplan',
  metadata: mockMetadata,
  author: { display: 'Joe Bloggs', type: 'Test' },
  intent: CarePlanIntentCode.Plan,
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  status: CarePlanStatusCode.Active,
  title: 'Test Care Plan Summary One',
}

const CarePlanTwo: CarePlan = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|careplan',
  metadata: mockMetadata,
  author: { display: 'John Mclane', type: 'Test' },
  intent: CarePlanIntentCode.Proposal,
  period: {
    start: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2015-03-04T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  status: CarePlanStatusCode.Completed,
  title: 'Test Care Plan Summary Two',
}

export { CarePlanOne, CarePlanTwo }
