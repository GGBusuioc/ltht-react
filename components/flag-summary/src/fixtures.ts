import { Flag, FlagStatusCode, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const FlagOne: Flag = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard',
  metadata: mockMetadata,
  author: {
    display: 'John Smith',
    reference:
      'https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser',
    type: 'Practitioner',
    identifier: undefined,
  },
  code: {
    coding: [{ display: 'Lives in Known High Risk Area', code: '109006' }],
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Year,
    },
  },
  status: FlagStatusCode.Inactive,
}

const FlagTwo: Flag = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard',
  metadata: mockMetadata,
  author: {
    display: 'John Smith',
    reference:
      'https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser',
    type: 'Practitioner',
    identifier: undefined,
  },
  code: {
    coding: [{ display: 'High Risk of Violence', code: '56706' }],
  },
  category: {
    coding: [{ display: 'Violence', code: '66706' }],
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },

  status: FlagStatusCode.Inactive,
}

export { FlagOne, FlagTwo }
