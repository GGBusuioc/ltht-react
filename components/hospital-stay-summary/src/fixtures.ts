import { HospitalStay, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const HospitalStayOne: HospitalStay = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|hospitalstay',
  metadata: mockMetadata,
  nhsNumber: '123456789',
  serviceProvider: { display: 'Chapel Allerton Hospital', type: 'Hospital' },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
}

const HospitalStayTwo: HospitalStay = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|hospitalstay',
  metadata: mockMetadata,
  nhsNumber: '987654321',
  serviceProvider: { display: 'Beaumont Hospital', type: 'Hospital' },
  period: {
    start: {
      value: '2020-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
}

export { HospitalStayOne, HospitalStayTwo }
