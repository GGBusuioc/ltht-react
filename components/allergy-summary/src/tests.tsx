import React from 'react'
import { mount } from 'enzyme'
import ReactDOM from 'react-dom'

import { AllergyIntolerance } from '@ltht-react/types'
import { AllergyOne, AllergyTwo } from './fixtures'
import AllergySummary from '.'

const handleClick = (allergy: AllergyIntolerance) => {}

describe('AllergySummary', () => {
  const allergies = [AllergyOne, AllergyTwo]
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<AllergySummary title="Allergies" allergies={allergies} clickHandler={handleClick} />, div)
  })
  it('matches snapshot', () => {
    expect(
      mount(<AllergySummary title="Allergies" allergies={allergies} clickHandler={handleClick} />)
    ).toMatchSnapshot('wrapper mount')
  })
})
