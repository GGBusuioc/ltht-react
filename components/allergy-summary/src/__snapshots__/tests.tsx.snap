// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllergySummary matches snapshot: wrapper mount 1`] = `
.emotion-20 {
  color: #231f20;
  font-family: 'SF Pro Text','SF Pro Icons','Helvetica Neue',Helvetica,Arial,sans-serif;
  font-size: 0.8rem;
  background: #FFFFFF;
  color: #231f20;
  margin-bottom: 0.5rem;
  border-radius: 4px;
  box-shadow: 3px 3px 3px #666666;
  margin: 0 5px 10px 0;
  -webkit-font-smoothing: antialiased;
}

.emotion-20 *,
.emotion-20 ::before,
.emotion-20 ::after {
  box-sizing: border-box;
  padding: 0;
  margin: 0;
}

.emotion-20 * {
  margin: 0;
  padding: 0;
}

.emotion-20 h3 {
  font-size: 1rem;
}

@media only screen and (min-width:768px) {
  .emotion-20 {
    font-size: 0.9rem;
  }

  .emotion-20 h3 {
    font-size: 1.1rem;
  }
}

@media only screen and (min-width:1024px) {
  .emotion-20 {
    font-size: 1rem;
  }

  .emotion-20 h3 {
    font-size: 1.2rem;
  }
}

@media only screen and (min-width:1200px) {
  .emotion-20 {
    font-size: 1.1rem;
  }

  .emotion-20 h3 {
    font-size: 1.3rem;
  }
}

.emotion-0 {
  padding: 0.5rem;
}

.emotion-19 {
  list-style: none;
  padding-left: 0.5rem;
}

.emotion-19 li {
  border-top: 1px solid #b0b0b0;
  padding: 0.5rem 0.5rem 0.5rem 0;
}

.emotion-19 li:hover {
  cursor: pointer;
  background: #E8EDEE;
}

.emotion-9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.emotion-3 {
  min-width: 1rem;
}

.emotion-1 {
  color: #DA291C;
}

.emotion-6 {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  padding-left: 0.5rem;
}

.emotion-4 {
  color: #231f20;
  text-align: left;
}

.emotion-5 {
  color: #768692;
  text-align: left;
}

.emotion-7 {
  color: #231f20;
  text-align: right;
}

.emotion-8 {
  color: #768692;
  text-align: right;
}

.emotion-10 {
  color: #768692;
}

<AllergySummary
  allergies={
    Array [
      Object {
        "assertedDate": Object {
          "kind": "YEAR",
          "value": "2013-01-01T00:00:00+00:00",
        },
        "category": Array [
          "ENVIRONMENT",
        ],
        "clinicalStatus": "ACTIVE",
        "code": Object {
          "coding": Array [
            Object {
              "code": "109006",
              "display": "Dust Mites",
            },
          ],
        },
        "criticality": "HIGH",
        "id": "580ca927-34e0-e911-a2c7-005056926fe4|allergyalert",
        "metadata": Object {
          "dataSources": Array [
            Object {
              "display": "Mock",
            },
          ],
        },
        "type": "ALLERGY",
        "verificationStatus": "CONFIRMED",
      },
      Object {
        "assertedDate": Object {
          "kind": "YEAR",
          "value": "2013-01-01T00:00:00+00:00",
        },
        "clinicalStatus": "RESOLVED",
        "code": Object {
          "coding": Array [
            Object {
              "code": "109007",
              "display": "Kiwi fruit",
            },
          ],
        },
        "id": "690ca927-68e0-c511-b2c7-898056926ze4|allergyalert",
        "metadata": Object {
          "dataSources": Array [
            Object {
              "display": "Mock",
            },
          ],
        },
        "type": "INTOLERANCE",
      },
    ]
  }
  clickHandler={[Function]}
  title="Allergies"
>
  <Widget>
    <div
      className="emotion-20"
    >
      <WidgetHeader>
        <div
          className="emotion-0"
        >
          <h3>
            Allergies
          </h3>
        </div>
      </WidgetHeader>
      <WidgetList
        clickable={true}
      >
        <ul
          className="emotion-19"
        >
          <WidgetListItem
            key="0"
          >
            <li>
              <AllergySummaryItem
                allergy={
                  Object {
                    "assertedDate": Object {
                      "kind": "YEAR",
                      "value": "2013-01-01T00:00:00+00:00",
                    },
                    "category": Array [
                      "ENVIRONMENT",
                    ],
                    "clinicalStatus": "ACTIVE",
                    "code": Object {
                      "coding": Array [
                        Object {
                          "code": "109006",
                          "display": "Dust Mites",
                        },
                      ],
                    },
                    "criticality": "HIGH",
                    "id": "580ca927-34e0-e911-a2c7-005056926fe4|allergyalert",
                    "metadata": Object {
                      "dataSources": Array [
                        Object {
                          "display": "Mock",
                        },
                      ],
                    },
                    "type": "ALLERGY",
                    "verificationStatus": "CONFIRMED",
                  }
                }
                clickHandler={[Function]}
              >
                <div
                  className="emotion-9"
                  onClick={[Function]}
                >
                  <div
                    className="emotion-3"
                  >
                    <AllergyIcon
                      allergy={
                        Object {
                          "assertedDate": Object {
                            "kind": "YEAR",
                            "value": "2013-01-01T00:00:00+00:00",
                          },
                          "category": Array [
                            "ENVIRONMENT",
                          ],
                          "clinicalStatus": "ACTIVE",
                          "code": Object {
                            "coding": Array [
                              Object {
                                "code": "109006",
                                "display": "Dust Mites",
                              },
                            ],
                          },
                          "criticality": "HIGH",
                          "id": "580ca927-34e0-e911-a2c7-005056926fe4|allergyalert",
                          "metadata": Object {
                            "dataSources": Array [
                              Object {
                                "display": "Mock",
                              },
                            ],
                          },
                          "type": "ALLERGY",
                          "verificationStatus": "CONFIRMED",
                        }
                      }
                    >
                      <ExclamationIcon
                        size="medium"
                        status="red"
                      >
                        <FontAwesomeIcon
                          border={false}
                          className="emotion-1"
                          fixedWidth={false}
                          flip={null}
                          icon={
                            Object {
                              "icon": Array [
                                576,
                                512,
                                Array [],
                                "f071",
                                "M569.517 440.013C587.975 472.007 564.806 512 527.94 512H48.054c-36.937 0-59.999-40.055-41.577-71.987L246.423 23.985c18.467-32.009 64.72-31.951 83.154 0l239.94 416.028zM288 354c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z",
                              ],
                              "iconName": "exclamation-triangle",
                              "prefix": "fas",
                            }
                          }
                          inverse={false}
                          listItem={false}
                          mask={null}
                          pull={null}
                          pulse={false}
                          rotation={null}
                          size="sm"
                          spin={false}
                          swapOpacity={false}
                          symbol={false}
                          title=""
                          transform={null}
                        >
                          <svg
                            aria-hidden="true"
                            className="svg-inline--fa fa-exclamation-triangle fa-w-18 fa-sm emotion-1"
                            data-icon="exclamation-triangle"
                            data-prefix="fas"
                            focusable="false"
                            role="img"
                            style={Object {}}
                            viewBox="0 0 576 512"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M569.517 440.013C587.975 472.007 564.806 512 527.94 512H48.054c-36.937 0-59.999-40.055-41.577-71.987L246.423 23.985c18.467-32.009 64.72-31.951 83.154 0l239.94 416.028zM288 354c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"
                              fill="currentColor"
                              style={Object {}}
                            />
                          </svg>
                        </FontAwesomeIcon>
                      </ExclamationIcon>
                    </AllergyIcon>
                  </div>
                  <div
                    className="emotion-6"
                  >
                    <AllergyTitle
                      allergy={
                        Object {
                          "assertedDate": Object {
                            "kind": "YEAR",
                            "value": "2013-01-01T00:00:00+00:00",
                          },
                          "category": Array [
                            "ENVIRONMENT",
                          ],
                          "clinicalStatus": "ACTIVE",
                          "code": Object {
                            "coding": Array [
                              Object {
                                "code": "109006",
                                "display": "Dust Mites",
                              },
                            ],
                          },
                          "criticality": "HIGH",
                          "id": "580ca927-34e0-e911-a2c7-005056926fe4|allergyalert",
                          "metadata": Object {
                            "dataSources": Array [
                              Object {
                                "display": "Mock",
                              },
                            ],
                          },
                          "type": "ALLERGY",
                          "verificationStatus": "CONFIRMED",
                        }
                      }
                    >
                      <div
                        className="emotion-4"
                      >
                        Dust Mites
                      </div>
                    </AllergyTitle>
                    <AllergyDescription
                      allergy={
                        Object {
                          "assertedDate": Object {
                            "kind": "YEAR",
                            "value": "2013-01-01T00:00:00+00:00",
                          },
                          "category": Array [
                            "ENVIRONMENT",
                          ],
                          "clinicalStatus": "ACTIVE",
                          "code": Object {
                            "coding": Array [
                              Object {
                                "code": "109006",
                                "display": "Dust Mites",
                              },
                            ],
                          },
                          "criticality": "HIGH",
                          "id": "580ca927-34e0-e911-a2c7-005056926fe4|allergyalert",
                          "metadata": Object {
                            "dataSources": Array [
                              Object {
                                "display": "Mock",
                              },
                            ],
                          },
                          "type": "ALLERGY",
                          "verificationStatus": "CONFIRMED",
                        }
                      }
                    >
                      <div
                        className="emotion-5"
                      >
                        109006 - Environment - Allergy
                      </div>
                    </AllergyDescription>
                  </div>
                  <div>
                    <AllergyDate
                      allergy={
                        Object {
                          "assertedDate": Object {
                            "kind": "YEAR",
                            "value": "2013-01-01T00:00:00+00:00",
                          },
                          "category": Array [
                            "ENVIRONMENT",
                          ],
                          "clinicalStatus": "ACTIVE",
                          "code": Object {
                            "coding": Array [
                              Object {
                                "code": "109006",
                                "display": "Dust Mites",
                              },
                            ],
                          },
                          "criticality": "HIGH",
                          "id": "580ca927-34e0-e911-a2c7-005056926fe4|allergyalert",
                          "metadata": Object {
                            "dataSources": Array [
                              Object {
                                "display": "Mock",
                              },
                            ],
                          },
                          "type": "ALLERGY",
                          "verificationStatus": "CONFIRMED",
                        }
                      }
                    >
                      <div
                        className="emotion-7"
                      >
                        2013
                      </div>
                    </AllergyDate>
                    <AllergyStatus
                      allergy={
                        Object {
                          "assertedDate": Object {
                            "kind": "YEAR",
                            "value": "2013-01-01T00:00:00+00:00",
                          },
                          "category": Array [
                            "ENVIRONMENT",
                          ],
                          "clinicalStatus": "ACTIVE",
                          "code": Object {
                            "coding": Array [
                              Object {
                                "code": "109006",
                                "display": "Dust Mites",
                              },
                            ],
                          },
                          "criticality": "HIGH",
                          "id": "580ca927-34e0-e911-a2c7-005056926fe4|allergyalert",
                          "metadata": Object {
                            "dataSources": Array [
                              Object {
                                "display": "Mock",
                              },
                            ],
                          },
                          "type": "ALLERGY",
                          "verificationStatus": "CONFIRMED",
                        }
                      }
                    >
                      <div
                        className="emotion-8"
                      >
                        Active - Confirmed
                      </div>
                    </AllergyStatus>
                  </div>
                </div>
              </AllergySummaryItem>
            </li>
          </WidgetListItem>
          <WidgetListItem
            key="1"
          >
            <li>
              <AllergySummaryItem
                allergy={
                  Object {
                    "assertedDate": Object {
                      "kind": "YEAR",
                      "value": "2013-01-01T00:00:00+00:00",
                    },
                    "clinicalStatus": "RESOLVED",
                    "code": Object {
                      "coding": Array [
                        Object {
                          "code": "109007",
                          "display": "Kiwi fruit",
                        },
                      ],
                    },
                    "id": "690ca927-68e0-c511-b2c7-898056926ze4|allergyalert",
                    "metadata": Object {
                      "dataSources": Array [
                        Object {
                          "display": "Mock",
                        },
                      ],
                    },
                    "type": "INTOLERANCE",
                  }
                }
                clickHandler={[Function]}
              >
                <div
                  className="emotion-9"
                  onClick={[Function]}
                >
                  <div
                    className="emotion-3"
                  >
                    <AllergyIcon
                      allergy={
                        Object {
                          "assertedDate": Object {
                            "kind": "YEAR",
                            "value": "2013-01-01T00:00:00+00:00",
                          },
                          "clinicalStatus": "RESOLVED",
                          "code": Object {
                            "coding": Array [
                              Object {
                                "code": "109007",
                                "display": "Kiwi fruit",
                              },
                            ],
                          },
                          "id": "690ca927-68e0-c511-b2c7-898056926ze4|allergyalert",
                          "metadata": Object {
                            "dataSources": Array [
                              Object {
                                "display": "Mock",
                              },
                            ],
                          },
                          "type": "INTOLERANCE",
                        }
                      }
                    >
                      <ExclamationIcon
                        size="medium"
                        status="default"
                      >
                        <FontAwesomeIcon
                          border={false}
                          className="emotion-10"
                          fixedWidth={false}
                          flip={null}
                          icon={
                            Object {
                              "icon": Array [
                                576,
                                512,
                                Array [],
                                "f071",
                                "M569.517 440.013C587.975 472.007 564.806 512 527.94 512H48.054c-36.937 0-59.999-40.055-41.577-71.987L246.423 23.985c18.467-32.009 64.72-31.951 83.154 0l239.94 416.028zM288 354c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z",
                              ],
                              "iconName": "exclamation-triangle",
                              "prefix": "fas",
                            }
                          }
                          inverse={false}
                          listItem={false}
                          mask={null}
                          pull={null}
                          pulse={false}
                          rotation={null}
                          size="sm"
                          spin={false}
                          swapOpacity={false}
                          symbol={false}
                          title=""
                          transform={null}
                        >
                          <svg
                            aria-hidden="true"
                            className="svg-inline--fa fa-exclamation-triangle fa-w-18 fa-sm emotion-10"
                            data-icon="exclamation-triangle"
                            data-prefix="fas"
                            focusable="false"
                            role="img"
                            style={Object {}}
                            viewBox="0 0 576 512"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M569.517 440.013C587.975 472.007 564.806 512 527.94 512H48.054c-36.937 0-59.999-40.055-41.577-71.987L246.423 23.985c18.467-32.009 64.72-31.951 83.154 0l239.94 416.028zM288 354c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"
                              fill="currentColor"
                              style={Object {}}
                            />
                          </svg>
                        </FontAwesomeIcon>
                      </ExclamationIcon>
                    </AllergyIcon>
                  </div>
                  <div
                    className="emotion-6"
                  >
                    <AllergyTitle
                      allergy={
                        Object {
                          "assertedDate": Object {
                            "kind": "YEAR",
                            "value": "2013-01-01T00:00:00+00:00",
                          },
                          "clinicalStatus": "RESOLVED",
                          "code": Object {
                            "coding": Array [
                              Object {
                                "code": "109007",
                                "display": "Kiwi fruit",
                              },
                            ],
                          },
                          "id": "690ca927-68e0-c511-b2c7-898056926ze4|allergyalert",
                          "metadata": Object {
                            "dataSources": Array [
                              Object {
                                "display": "Mock",
                              },
                            ],
                          },
                          "type": "INTOLERANCE",
                        }
                      }
                    >
                      <div
                        className="emotion-4"
                      >
                        Kiwi fruit
                      </div>
                    </AllergyTitle>
                    <AllergyDescription
                      allergy={
                        Object {
                          "assertedDate": Object {
                            "kind": "YEAR",
                            "value": "2013-01-01T00:00:00+00:00",
                          },
                          "clinicalStatus": "RESOLVED",
                          "code": Object {
                            "coding": Array [
                              Object {
                                "code": "109007",
                                "display": "Kiwi fruit",
                              },
                            ],
                          },
                          "id": "690ca927-68e0-c511-b2c7-898056926ze4|allergyalert",
                          "metadata": Object {
                            "dataSources": Array [
                              Object {
                                "display": "Mock",
                              },
                            ],
                          },
                          "type": "INTOLERANCE",
                        }
                      }
                    >
                      <div
                        className="emotion-5"
                      >
                        109007 - Intolerance
                      </div>
                    </AllergyDescription>
                  </div>
                  <div>
                    <AllergyDate
                      allergy={
                        Object {
                          "assertedDate": Object {
                            "kind": "YEAR",
                            "value": "2013-01-01T00:00:00+00:00",
                          },
                          "clinicalStatus": "RESOLVED",
                          "code": Object {
                            "coding": Array [
                              Object {
                                "code": "109007",
                                "display": "Kiwi fruit",
                              },
                            ],
                          },
                          "id": "690ca927-68e0-c511-b2c7-898056926ze4|allergyalert",
                          "metadata": Object {
                            "dataSources": Array [
                              Object {
                                "display": "Mock",
                              },
                            ],
                          },
                          "type": "INTOLERANCE",
                        }
                      }
                    >
                      <div
                        className="emotion-7"
                      >
                        2013
                      </div>
                    </AllergyDate>
                    <AllergyStatus
                      allergy={
                        Object {
                          "assertedDate": Object {
                            "kind": "YEAR",
                            "value": "2013-01-01T00:00:00+00:00",
                          },
                          "clinicalStatus": "RESOLVED",
                          "code": Object {
                            "coding": Array [
                              Object {
                                "code": "109007",
                                "display": "Kiwi fruit",
                              },
                            ],
                          },
                          "id": "690ca927-68e0-c511-b2c7-898056926ze4|allergyalert",
                          "metadata": Object {
                            "dataSources": Array [
                              Object {
                                "display": "Mock",
                              },
                            ],
                          },
                          "type": "INTOLERANCE",
                        }
                      }
                    >
                      <div
                        className="emotion-8"
                      >
                        Resolved
                      </div>
                    </AllergyStatus>
                  </div>
                </div>
              </AllergySummaryItem>
            </li>
          </WidgetListItem>
        </ul>
      </WidgetList>
    </div>
  </Widget>
</AllergySummary>
`;
